build.sh

```diff
@@ -193,6 +193,8 @@ fi
 if hasArg libcudf_kafka; then
     cmake -S $REPODIR/cpp/libcudf_kafka -B ${KAFKA_LIB_BUILD_DIR} \
           -DCMAKE_INSTALL_PREFIX=${INSTALL_PREFIX} \
+          -DBUILD_TESTS=${BUILD_TESTS} \
+          -DCUDF_USE_ARROW_STATIC=OFF \
```

Comment:
Do we need this option here? I believe this is the default and we'd want to follow the default in this script.

Reply:
Honestly not sure? Let me try removing it and see what happens.

Reply:
We may still need the `-DBUILD_TESTS=${BUILD_TESTS}` since `libcudf_kafka` does have some gtests from the looks of it?

Reply:
Ok, added it back.